src/main/java/uk/gov/companieshouse/api/accounts/service/impl/DebtorsService.java

```diff
@@ -42,10 +42,9 @@
     private DebtorsValidator debtorsValidator;
 
     @Autowired
-    public DebtorsService(DebtorsRepository repository,
-                          DebtorsTransformer transformer,
-                          SmallFullService smallFullService,
-                          KeyIdGenerator keyIdGenerator, DebtorsValidator debtorsValidator) {
+    public DebtorsService(DebtorsRepository repository, DebtorsTransformer transformer,
+                          SmallFullService smallFullService, KeyIdGenerator keyIdGenerator,
```

Comment:
I would indent it to 4 spaces as suggest in the coding standards: 
https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent

Reply:
indentation corrected in bdb836b